package.json

```diff
@@ -4,12 +4,13 @@
   "repository": "https://github.com/dimagi/commcare-hq",
   "description": "CommcareHQ Javascript tests",
   "dependencies": {
+    "eslint-plugin-eslint-dimagi": "file:eslint",
```

Comment:
A lot of these should probably be `devDependencies` (probably everything except for `js-xpath` and `uglifyjs`